index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <div class="container">
+        <header>
+            <div class="right-side">
+                <h3>Anicet Fantômas</h3>
+            </div>
+            <div class="left-side">
+                <div class="menu-lines">
+                    <div class="menu-line"></div>
+                    <div class="menu-line"></div>
+                    <div class="menu-line"></div>
+                </div>
+            </div>
+        </header>
+        <section>
+            <h1>Hey There.<br>I'm Anicet</h1>
+            <h2>I'am a Software Developer</h2>
+            <p>I can help you build a product. featuer or <br> website Look through some of my work <br> and experience!
+                if you see <br>
+                and have a project you need coded. <br> don't hestiate to contact me.
+            </p>
```

Comment:
- [ ] Did you know that you can find all the style details in figma if you click on elements? If no then lets try to do it! click on element and check the right section. You will see all the specifications like font size, colors etc. 
Please try to check the styles for heading and paragraph of the page and apply them on your project!

Figma   |  Yours
------------- | -------------
<img width="381" alt="Screenshot 2022-02-10 at 22 16 55" src="https://user-images.githubusercontent.com/77038610/153471044-033a9880-daa0-4454-b9a2-1a7d834f7a2f.png">| <img width="418" alt="Screenshot 2022-02-10 at 22 16 01" src="https://user-images.githubusercontent.com/77038610/153470901-de1853c3-7a86-48aa-b45a-636b170e65a1.png">


Please set the same colors and font styles 😉